vault/expiration.go

```diff
@@ -1187,6 +1250,10 @@ func (m *ExpirationManager) loadEntryInternal(leaseID string, restoreMode bool,
 			}
 		}
 
+		if le.Version < 2 {
+			atomic.CompareAndSwapInt32(&m.SHA1HashedLeasesPresent, 0, 1)
```

Comment:
It might be easier (and faster) to just do a StoreInt32 rather than a compare and swap

Reply:
Fixed.